pipeline/relation_embedder/router/src/main/scala/uk/ac/wellcome/platform/router/RouterWorkerService.scala

```diff
@@ -26,8 +27,17 @@ class RouterWorkerService[MsgDestination](
   private def processMessage(message: NotificationMessage): Future[Unit] = {
     workRetriever.apply(message.body).flatMap { work =>
       work.data.collectionPath
-        .fold(ifEmpty = Future.fromTry(worksMsgSender.sendT(work.id))) { path =>
-          Future.fromTry(pathsMsgSender.sendT(path))
+        .fold(ifEmpty = {
+          for {
+           worksEither <- workIndexer.index(work.transition[Denormalised] (Relations.none))
```

Comment:
:thinking: it might actually be worth using `groupedWithin` like we do elsewhere to batch up the works before indexing, as we have encountered issues previously with ES performance.

Reply:
We don't actually do that everywhere on second thoughts, but e.g. in the merger we do send the whole merged works in one go rather than individually.

Reply:
yeah, I realised this too. This isn't batching and it will probably cause problems. I feel that if we're going to use the pipeline storage on other pipeline services it should be somehow provided by that though. Most services in the pipeline work in a "one in one out" way like the router. The merger and the relation embedder are exceptions in this sense and it might be worth to do it in the merger too anyway.